internal/cluster/transition.go

```diff
@@ -524,10 +525,50 @@ func (th *transitionHandler) PostPreparingTimedOut(sw stateswitch.StateSwitch, a
 		updatedCluster *common.Cluster
 	)
 
+	// Capture some information about hosts states, we should be able to determine more about the cause of the timeout.
+	// This can occur because the host...
+	//
+	// 1: Becomes disconnected - the "connected" validation of the host will be failing
```

Comment:
disconnected is handled by another transition https://github.com/openshift/assisted-service/blob/2c70724e4b5f79c472847a0d78dcd77f0f2a0197/internal/host/statemachine.go#L696C1-L710